### DISCUSSION TITLE
How to add paging to data grid?

### DISCUSSION THREAD

**Discussion by ahmed-adly-khalil** [microsoft/fluentui#25899] [Q&A]
Hello,
is there a way to add paging to Data Grid react component: https://react.fluentui.dev/?path=/docs/preview-components-datagrid--default ? 

Any ideas would be highly appreciated

**Comment by JustSlone** [comment]
@ling1726 do we support paging or have a way to add on paging for DataGrid yet?

**Reply by ahmed-adly-khalil** [reply]
@ling1726 any ideas here? thanks so much

**Reply by ling1726** [reply]
Hi, pagination state is a feature that is already prototyped, However, due to our capacity will only be delivered during Q1 2023. In the meantime, you can achieve pagination simply by controlling the state externally from the component.

Also Fluent v9 does not currently have a pagination component, so maybe @JustSlone  or @paulgildea  might provide some information on that
